modules/4337/contracts/Safe4337Module.sol

```diff
@@ -211,6 +211,42 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
         operationHash = keccak256(operationData);
     }
 
+    /**
+     * @dev Checks if the signatures length is correct and does not contain addtional bytes.
+     * @param signatures signatures data
```

Comment:
nit: missing explicit documentation that this does not check integrity of encoding, as the Safe.checkSignatures already does this.

Reply:
https://github.com/safe-global/safe-modules/pull/453/commits/9e526d6de72915fcafc4fb4b0a9f1013821656fc